libs/execution_context/ebpf_native.c

```diff
@@ -0,0 +1,1219 @@
+// Copyright (c) Microsoft Corporation
```

Comment:
Can you add function entry / exit tracing and log any error path that user behavior might trigger?

Reply:
Done.